R/fit_DoseResponseCurve.R

```diff
@@ -246,10 +246,10 @@
 #' `HPDI68_U` \tab [numeric] \tab same as `HPDI68_L` for the upper bound \cr
 #' `HPDI95_L` \tab [numeric] \tab same as `HPDI68_L` but for 95% probability \cr
 #' `HPDI95_U` \tab [numeric] \tab same as `HPDI95_L` but for the upper bound \cr
-#'
+#' `De.raw` \tab [numeric] \tab only for `mode = "interpolation"`; reports the fitted De values (can also be negative)\cr
```

Comment:
Here we should be careful. It sounds lake that we are filtering the other values. We should better write: "Reports all calculated values 'as is' without setting meaningless values to `NA`. In particular it reports `Inf` (`-Inf`) and negative values if they could be calculated (`NaN` values would be still `NA`).